parsers/GB.py

```diff
@@ -25,20 +25,11 @@ def fetch_price(
     target_datetime: Optional[datetime] = None,
     logger: Logger = getLogger(__name__),
 ) -> list:
-    if target_datetime:
```

Comment:
If we no longer can fetch historical data we should raise a not Implemented exception when a datetime is passed to the function.

Reply:
Your comment made me looking for how rte-france is fetching historical data, they use another URL. We only need to change this URL in the code.